### PR TITLE
NC-3

### PR DESCRIPTION
**Pull request by starikandrei**

### CODE REVIEW

File: blocklearning/lib/main.dart [hunk 1]
```diff
@@ -32,10 +33,7 @@ Future<void> main() async {
         GlobalCupertinoLocalizations.delegate,
       ],
       translationsKeys: AppTranslation.translations,
-      theme: ThemeData(
-        useMaterial3: true,
-        scaffoldBackgroundColor: AppColors.background,
-      ),
+      theme: baseTheme,
```
Comment (dadsparta) [inline blocklearning/lib/main.dart:41]: Why did you move theme in other file?
I think, that's good, but we have special file for 1 theme... 
Is it really good for architecture?
Maybe we need to leave theme in main, because it's using only in this place and we always can change it here

**Review by axpreparator — APPROVED** [review]

**Review by dadsparta — APPROVED** [review]
